config/database.yml

```diff
@@ -23,4 +23,10 @@ production:
   adapter: postgresql
   pool: 5
   timeout: 5000
-  database: bjc_teachers_prod
\ No newline at end of file
+  database: bjc_teachers_prod
+
+production:
```

Comment:
Shouldn’t this key be staging?

Reply:
Oh I think I got prod and stage database name mixed up.